src/snackBar/Main.java

```diff
@@ -0,0 +1,75 @@
+package snackBar;
+
+public class Main {
+
+    public static void workWithData() {
+        Customer c1 = new Customer("Jane", 45.25);
+        Customer c2 = new Customer("Bob", 33.14);
+
+        VendingMachine v1 = new VendingMachine("Food");
+        VendingMachine v2 = new VendingMachine("Drink");
+        VendingMachine v3 = new VendingMachine("Office");
+
+        Snack s1 = new Snack("Chips", 36, 1.75, v1.getId());
+        Snack s2 = new Snack("Chocolate Bar", 36, 1.00, v1.getId());
+        Snack s3 = new Snack("Pretzel", 30, 2.00, v1.getId());
+
+        Snack s4 = new Snack("Soda", 24, 2.50, v2.getId());
+        Snack s5 = new Snack("Water", 20, 2.75, v2.getId());
+
+        // Problem 1
+        c1.buyItem(3 * s4.getCost());
+        s4.buySnack(3);
+
```

Comment:
Nice job getting MVP. One small suggestion: maybe add some blank lines in your console or labels to organize it better. something like    `  System.out.println("*** Customer 1 buys 3 of snack 4. ***");`